contents/docs/_snippets/how-to-link-events-to-groups.mdx

```diff
@@ -12,6 +12,8 @@ import JSGroupEventCodeSnippet from "../\_snippets/capture-group-event-code.mdx"
 
 > **Want to learn more about group analytics implementation differences?** Check out our [guide on frontend vs backend group analytics implementations](/tutorials/frontend-vs-backend-group-analytics).
 
+> **Identified events:** Events must be [identified](/docs/data/anonymous-vs-identified-events) in order to be associated with a group. If the `$process_person_profile` event property ends up being set to `false`, the event will not be associated with the group.
```

Comment:
I think this can be a normal paragraph (mostly because this is the right location but we shouldn't have two quotes in a row)
```suggestion
Events must be [identified](/docs/data/anonymous-vs-identified-events) to be associated with a group. If the `$process_person_profile` event property ends up being set to `false`, the event will not be associated with the group.
```